src/initialization.jl

```diff
@@ -91,7 +91,10 @@ function __init__()
         thread_streams[tid] = nothing
     end
 
-    @require ForwardDiff="f6369f11-7733-5829-9624-2563aa707210" include("forwarddiff.jl")
+    precompiling = ccall(:jl_generating_output, Cint, ()) != 0
+    if !precompiling
+        eval(overrides)
```

Comment:
`empty!(overrides)`?

Reply:
Particularly motivated by me wanting to do overrides in an outside package, calling this twice shouldn't hurt.

Reply:
without the `empty!` you get:

```
vchuravy@cyclops:~/dev/KernelAbstractions$ ~/julia-1.6.0-rc1/bin/julia --project=lib/CUDAKernels mt.jl 
ERROR: LoadError: InitError: UndefVarError: FastMath not defined
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/GPUCompiler/gbnMW/src/jlgen.jl:156 [inlined]
 [2] top-level scope
   @ ~/.julia/packages/CUDA/9IRgE/src/device/intrinsics.jl:7
 [3] eval
   @ ./boot.jl:360 [inlined]
 [4] eval
   @ ~/.julia/dev/KernelAbstractions/mt.jl:1 [inlined]
 [5] __init__()
   @ Main.CUDAKernels ~/.julia/dev/KernelAbstractions/mt.jl:9
during initialization of module CUDAKernels
in expression starting at /home/vchuravy/.julia/dev/KernelAbstractions/mt.jl:1
```

Reply:
If you do overrides from an external package, you just want to `eval` those yourself? Calling `__init__` twice is likely going to break in other cases (double-registering hooks, re-initializating thread local caches, etc).